src/Controls/tests/UITests/Tests/Issues/Issue17789.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue17789 : _IssuesUITest
+	{
+		public Issue17789(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "ContentPage BackgroundImageSource not working";
+
+		[Test]
+		public void ContentPageBackgroundImageSourceWorks()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.Windows },
+					"The bug only happens on iOS; see https://github.com/dotnet/maui/pull/17789");
```

Comment:
It only happens on iOS _right now_ - if you enable the test on all the platforms, we can prevent it happening elsewhere later.

Reply:
Right, changed it to pass the tests in all the platforms. Waiting the build to generate the snapshots by platform and include it in the PR.

Reply:
Added pending snapshots.